python/tvm/relay/op/contrib/tensorrt.py

```diff
@@ -134,15 +135,18 @@ def partition_for_tensorrt(
 
     if params:
         mod["main"] = bind_params_by_name(mod["main"], params)
+
     seq = tvm.transform.Sequential(
         [
             transform.InferType(),
             RemoveDropoutPass(),
             transform.RemoveUnusedFunctions(),
             transform.ConvertLayout(
-                {"nn.conv2d": ["NCHW", "default"], "nn.conv3d": ["NCDHW", "default"]}
+                {"nn.conv2d": ["NCHW", "default"],
+                 "nn.conv3d": ["NCDHW", "default"]}
             ),
             transform.FoldConstant(),
+            transform.InferType(),
```

Comment:
Do we need this?

Reply:
Thanks for pointing, we don't, was used for debugging purposes only.